lib/openscap_parser/value_definition.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+module OpenscapParser
+  class ValueDefinition < XmlNode
+    include OpenscapParser::Util
+
+    def id
+      @id ||= parsed_xml['id']
+    end
+
+    def description
+      @description ||= newline_to_whitespace(parsed_xml.at_css('description')&.text)
+    end
+
+    def title
+      @title ||= parsed_xml.at_css('title')&.text
+    end
+
+    def type
+      @type ||= parsed_xml['type'] || 'string'
+    end
+
+    def lower_bound
+      lower_bound_element = parsed_xml.at_xpath("lower-bound[@selector='']") || parsed_xml.at_xpath('lower-bound[not(@selector)]')
+      @lower_bound ||= lower_bound_element&.text
+    end
+
+    def upper_bound
+      upper_bound_element = parsed_xml.at_xpath("upper-bound[@selector='']") || parsed_xml.at_xpath('upper-bound[not(@selector)]')
+      @upper_bound ||= upper_bound_element&.text
+    end
+
+    def default_value
+      # The default value is the value element with a empty or absent @selector
+      # If there is no value element with an empty or absent @selector, the first value in
+      # the top down processing shall be the default element
+      value_element = parsed_xml.at_xpath("value[@selector='']") || parsed_xml.at_xpath('value[not(@selector)]') || parsed_xml.xpath("value")[0]
+      @default_value ||= value_element&.text
```

Comment:
The point of [memoization](https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/) is to avoid calculation when the method is called for the second time:
```suggestion
      @default_value ||= begin
        value_element = parsed_xml.at_xpath("value[@selector='']") || parsed_xml.at_xpath('value[not(@selector)]') || parsed_xml.xpath("value")[0]
        value_element&.text
      end
```

Reply:
Ah right, this makes sense. I'll update it!